tekton/integration_pipeline.go

```diff
@@ -349,6 +349,11 @@ func (r *IntegrationPipelineRun) WithSnapshot(snapshot *applicationapiv1alpha1.S
 		_ = metadata.CopyLabelsByPrefix(&snapshot.ObjectMeta, &r.ObjectMeta, prefix)
 	}
 
+	// If the Annotation is PaC log-url, we need to set the value to the console URL of the PipelineRun
+	if metadata.HasAnnotation(r, PipelinesAsCodePrefix+"/log-url") {
+		pipelineRunConsoleURL := "https://console.redhat.com/openshift/pipelines/ns/" + r.Namespace + "/pipelineruns/" + r.Name
```

Comment:
The base URL is different for different kinds of clusters (staging, internal, user-provisioned, etc.), so I think we should use the [FormatPipelineRunURL](https://github.com/konflux-ci/integration-service/blob/1025b17955efd24a334e549bba327db7c72ad0e5/status/format.go#L248) function which accounts for these differences.

Reply:
when I tried to use it ,  I've added an import of  `"github.com/konflux-ci/integration-service/status"`
but I got some circle conflict with pkges 
maybe I'm using it incorrectly